pkg/containerd/events.go

```diff
@@ -180,14 +189,49 @@ func (e *EventMonitor) Close() error {
 	return e.containerdClient.Close()
 }
 
+func (e *EventMonitor) initializeExistingContainers(ctx context.Context) {
+	//nolint:godox
+	// TODO (Nino-k): add filters to only get a list of running containers
+	// currently there is no documentation on how the filters work
+	// e.g. []string{"spec.Status.State==running"}
+	runningContainers, err := e.containerdClient.ContainerService().List(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, container := range runningContainers {
```

Comment:
I could not find the container status on the Container's [Runtime](https://github.com/containerd/containerd/blob/e14dca4a40e4cc28364ddfa2352f3d062977e9cc/containers/containers.go#L50) / [RuntimeInfo](https://github.com/containerd/containerd/blob/e14dca4a40e4cc28364ddfa2352f3d062977e9cc/containers/containers.go#L87). Also, the OCI Runtime [spec](https://github.com/opencontainers/runtime-spec/blob/494a5a6aca782455c0fbfc35af8e12f04e98a55e/specs-go/config.go#L6) which is used as an object to unmarshall is not including any status info nor there is clear documentation. However, as a workaround, we check for the key `nerdctl/ports` in labels since that is only available if containers is running or `status` is `up`.